apb.yml

```diff
@@ -43,4 +43,19 @@ plans:
       default: "55"
       enum: ['55', '56']
       type: enum
-      title: MySQL version
\ No newline at end of file
+      title: MySQL version
```

Comment:
@pb82 just curious - does it work the other way around too ? 

From a `MAR` (e.g. Android app), binding to the push service ?